addons/website/views/snippets/s_map.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template name="Map" id="s_map">
+    <section class="s_map" style="min-height: 100px;" data-map-type="m" data-map-zoom="12"  data-map-min-height="100px"
+        t-att-data-map-address="' '.join(filter(None, (user_id.company_id.street, user_id.company_id.city, user_id.company_id.state_id.display_name, user_id.company_id.country_id.display_name)))"
+    >
+        <div class="map_container o_not_editable">
+            <div class="css_non_editable_mode_hidden">
+                <div class="missing_option_warning alert alert-info rounded-0 fade show d-none d-print-none">
+                    An address must be specified for a map to be embedded
+                </div>
+            </div>
+            <iframe class="d-none o_embedded o_not_editable" width="100%" height="100%" src="about:blank" frameborder="0" scrolling="no" marginheight="0" marginwidth="0"></iframe>
+            <div class="o_we_bg_filter" style="background-color: rgba(239, 112, 112, 0) !important;"></div>
```

Comment:
You reused the class of another web_editor option but you recode its whole behavior and add it in a different place in the DOM... that can only be a source of mistake :/ Is not it possible to add it at the same place as the normal color filter option and to not re-code the behavior?

Reply:
Sorry, I fail to understand this comment.
What is the re-coded behavior?
I could use another class name and define the adequate CSS, but I do not think this is what you suggest.
The only "normal color filter option" I see is about the background color of images (but it comes in a bundle with many other options), is this the one you have in mind?

Reply:
This option (and all the code related to o_we_bg_filter class):
![Screenshot from 2021-06-17 14-23-23](https://user-images.githubusercontent.com/10338094/122395998-d7973180-cf77-11eb-9181-265da1266013.png)

Here you re-code that option (there is a color filter option in your UI targeting that div) and at the same time you use part of the code of the original option (the `o_we_bg_filter` class) but not using it the same way as the original option (as you put that div in a different location in the DOM). Either fully re-code the option with a different class (that would be a bit sad but at least it would be an independent self-working feature of the map snippet which thus will never conflict with any change/migration done to the base filter option) or, better, use the existing color filter option the way it is meant to be used and without any new code. Actually, there is probably code which is in charge of excluding the map snippet from receiving the background options (color, image, video, filter, shape). I wonder why we did this in the first place: except "video", all could make sense. Maybe worth blaming the code to see why we disabled those on map snippet... and if no explanation found, worth checking with SBU? In any case, your code here should be about enabling at least the filter option and removing the custom one I guess.